app/controllers/StripeController.scala

```diff
@@ -129,7 +133,11 @@ class StripeController(paymentServices: PaymentServices, stripeConfig: Config, o
         .addingToSession("amount" -> contributionAmount.show)
         .setCookie[ContribTimestampCookieAttributes](Instant.ofEpochSecond(charge.created).toString)
     }.recover {
-      case e: Stripe.Error => BadRequest(Json.toJson(e))
+      case e: Stripe.Error => {
```

Comment:
Thats a good idea, but I think we should do it in a different PR. Remember the aim of this PR is to get the logging and metrics out without changing any functionality, so that we minimise the risk of deploying it.